core/src/main/java/com/github/mizool/core/rsa/RsaKeys.java

```diff
@@ -0,0 +1,96 @@
+package com.github.mizool.core.rsa;
+
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.interfaces.RSAPrivateCrtKey;
+import java.security.interfaces.RSAPrivateKey;
+import java.security.interfaces.RSAPublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.KeySpec;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.security.spec.RSAPublicKeySpec;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+import java.util.regex.Pattern;
+
+import lombok.experimental.UtilityClass;
+
+import com.github.mizool.core.exception.ConfigurationException;
+
+@UtilityClass
+public class RsaKeys
+{
+    private static final Pattern HEADER_FOOTER_AND_LINE_BREAKS = Pattern.compile(
```

Comment:
Oh, right. This is a @UtilityClass. Then remove the static here as well.

Reply:
I already had the same idea. However, when I removed the keyword, Sonar started to complain that the result of `Pattern.compile` should always be assigned to a `static final` field. Looks like the sonar rule does not know about Lombok.
I'm afraid that is a conflict of interest with [OMG-29](https://incub8.atlassian.net/browse/OMG-29). I'm not sure which route we should take with this.

I think, I'll remove the keyword here, giving our decision in the OMG precedence, we mark this as false positive in Sonar and talk about this during one of our next meetings.

Reply:
I've created [CCCP-34](https://incub8.atlassian.net/browse/CCCP-34) to further this discussion.